Composer/packages/client/src/recoilModel/selectors/diagnosticsPageSelector.ts

```diff
@@ -78,22 +76,14 @@ export const diagnosticsSelector = selectorFamily({
     diagnostics.forEach((d) => {
       diagnosticList.push(new BotDiagnostic(rootProjectId, projectId, '', d.source, d));
     });
-    BotIndexer.validate(botAssets).forEach((d) => {
-      diagnosticList.push(new BotDiagnostic(rootProjectId, projectId, '', d.source, d));
-    });
-    const skillDiagnostics = BotIndexer.checkSkillSetting(botAssets);
-    skillDiagnostics.forEach((item) => {
+
+    BotIndexer.validate(botAssets).forEach((item) => {
       if (item.source.endsWith('.json')) {
-        diagnosticList.push(new SkillDiagnostic(rootProjectId, projectId, item.source, item.source, item));
+        diagnosticList.push(new BotDiagnostic(rootProjectId, projectId, '', item.source, item));
       } else {
         diagnosticList.push(new DialogDiagnostic(rootProjectId, projectId, item.source, item.source, item));
       }
     });
-    const luisLocaleDiagnostics = BotIndexer.checkLUISLocales(botAssets);
```

Comment:
Can this be removed? The warning to indicate that LUIS is not supported in a region would not show up without this right. In addition, how did u get rid of the duplicates in the list?